general/1_terminal/Serhii-Solovei/Answers tasks 1,2,4.md

```diff
@@ -0,0 +1,14 @@
+Task_1: If you wanna find file/files in directory/directories by the name or part of the name - 
+	you can use find /way to directory(be default its current directory)/ -name "zoola*".
+
+Task_2: The working directory didn't change because command "cd $1" tries to change current directory on "dir", 
+	but there aren't any files with that name. 
+	As a result the code runs again and create one more directory with name "dir" inside the created "nested" directory. 
+	Flag -p create nested directories, if they don't exist already.
```

Comment:
Not quite correct.

Let's insert `ls -l` between `mkdir and cd`  and insert `pwd` after `cd` to see what is really happening.

Reply:
Yes, thanks. now I understand what is the problem. I've already add a new commit.